test/lakefsfs/build.sbt

```diff
@@ -0,0 +1,12 @@
+name := "lakefsfs"
+
+version := "0.1"
+
+scalaVersion := "2.12.10"
+
+libraryDependencies ++= Seq(
+    "org.apache.spark" %% "spark-sql" % "3.1.1" % "provided",
```

Comment:
Do we want to go Spark 3.1.1?  I guess we'll see when we ask customers to run it :-)

Reply:
Had to run it locally
But I'll check if there is a common way to generate a multi-target solution.

Reply:
@arielshaqed updated the build to enable the above.